packages/cli/docsgen/markdown.ts

```diff
@@ -90,7 +90,7 @@ function renderExamplesSection(examples: CliExample[], sectionTitle?: string, lo
 function renderOption(optionName: string, option: CliOptionDefinition): string | undefined {
   if (option.hidden) return;
 
-  const commandOption = [`<h3 id='-${optionName}'><code>--${optionName}</code><a href="#-${optionName}" class="hash-link" title="--${optionName}"></a></h3>`];
+  const commandOption = [`### \`--${optionName}\` {#-${optionName}}`];
```

Comment:
The cli flag headers look much bigger than what we currently have

Reply:
looks like we add 4x # previously

https://github.com/ChainSafe/lodestar/blob/c3afb0beaaa5ba9ced4c2c6c68202fecedb18ef3/packages/cli/docsgen/markdown.ts#L89